fplc_peak_analysis.py

```
@@ -0,0 +1,477 @@
+import numpy as np
+import logging
+from scipy.signal import find_peaks
+from dataclasses import dataclass, field
```

Ruff (0.12.2)

`dataclasses.field` imported but unused

Remove unused import: `dataclasses.field`

(F401)